composer.json

```diff
@@ -34,7 +34,8 @@
         "phpunit/phpunit": "^4.7|^5.0",
         "squizlabs/php_codesniffer": "^2.3",
         "jakub-onderka/php-parallel-lint": "^0.9.0",
-        "satooshi/php-coveralls": "^0.6.1"
+        "satooshi/php-coveralls": "^0.6.1",
+        "doctrine/orm": "^2.5"
```

Comment:
Is it possible to require ramsey/uuid-doctrine and somehow doctrine/orm not get pulled in by Composer by using our doctrine/dbal requirement? If so, maybe doctrine/orm should be moved up to the standard requirements for this package.